samples/java-protobuf-customer-registry-kafka-quickstart/docker-compose.yml

```diff
@@ -13,31 +13,38 @@ services:
         -Dkalix.proxy.eventing.support=kafka
       USER_FUNCTION_HOST: ${USER_FUNCTION_HOST:-host.docker.internal}
       USER_FUNCTION_PORT: ${USER_FUNCTION_PORT:-8080}
-      BROKER_CONFIG_FILE: /conf/my-local.kafka.properties
-    volumes:
-      - .:/conf
+      # configuring kafka broker used for eventing
+      BROKER_SERVERS: kafka:29092
+      # Alternatively:
+      # - for easy local/dev mode - plaintext, for non dev-mode TLS, use:
+      #BROKER_AUTH_MECHANISM: "PLAIN"
+      #BROKER_USERNAME: [your-username]
+      #BROKER_PASS: [your-pass]
```

Comment:
Note that for TLS they would both have to map those certs into Kafka and configure it to use them and also have to mount the CA cert into the proxy and configure it to use that in the truststore for the Kafka client, not entirely sure we should mention it. Or if we do we should document all of that.

Reply:
Yeah, I had the same questions, if we needed to actually have these different options here. I ended up adding them because we previously also had this `my-dev.kafka.properties` using user/pass... so I thought we better keep a similar example with the new approach.. But I'm not really sure. You've been working on this integration, I think I will just follow your whatever you feel is best.. should we just remove and keep this simple? Leave the details for docs?

Reply:
Let's drop and don't mention and follow up the day there is a user/customer who wants to run integration tests with credentials/TLS.